File: NgocNhanShop.Angular.AdminApp/ClientApp/src/app/models/action/controller-name-option.ts
import { ActionNameOption } from "./action-name-option";

export interface IControllerNameOption {
    
  controllerName: string;
  actionNameOptions:ActionNameOption[];
}

export class ControllerNameOption {

    controllerName: string;
    actionNameOptions:ActionNameOption[];

    constructor(data:IControllerNameOption){
      this.controllerName = data.controllerName;
      this.actionNameOptions = data.actionNameOptions;
    }
}

File: NgocNhanShop.ViewModel/System/Actions/Dtos/ControllerNameOption.cs
using NgocNhanShop.Data.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace NgocNhanShop.ViewModel.System.Actions.Dtos
{
    public class ControllerNameOption
    {
        public string ControllerName { get; set; }
        public List<ActionNameOption> ActionNameOptions { get; set; }
    }
}
